keanu/app.py

```diff
@@ -37,6 +37,9 @@ def before_request() -> tuple:
     Checks if a token header in requests
     :return:
     """
+    flask_app.logger.debug('Headers: %s', request.headers)
```

Comment:
Is it possible to change `flask_app.logger.debug` to `flask_app.logger.log` so it will still log when I turn debug off